### PR TITLE
Tests added

### PR DESCRIPTION
**Pull request by TomasVotruba**
**Test status - OK**: https://travis-ci.org/TomasVotruba/nette-identity-doctrine

Test suite runs for PHP 5.3-7, includes Nette 2.1, 2.2 and 2.3 (all three only for PHP 5.6, no need to test Nette for PHP versions).

(PHP 7 currently fails due to some DateTime changes.)

### Among other changes:

- bump min version to Nette 2.1, since Nette 2.0 is in [EOL since 2015-01](http://nette.org/en/download#toc-previous-versions)

- bump PSR-0 to PSR-4, since [PSR-0 is deprecated since 2014-10](http://www.php-fig.org/psr/psr-0/)

- includes Nette DI fix: https://github.com/Majkl578/nette-identity-doctrine/pull/5/files, https://github.com/Majkl578/nette-identity-doctrine/pull/7/files

- simpler README, more clear entity (with PHP 5.3 syntax and annotations)

### CODE REVIEW

**Comment by TomasVotruba** [comment]
@Majkl578 Should be ready now. Please check.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  include:
+    - php: 5.6
+      evn: NETTE_VERSION=2.1.*
+    - php: 5.6
+      evn: NETTE_VERSION=2.2.*
+  allow_failures:
+    - php: 7.0
+
+install:
+  - sh -c 'if [ "$NETTE_VERSION" != "" ]; then composer require --no-update symfony/symfony=$NETTE_VERSION; fi;'
```
Comment (TomasVotruba) [inline .travis.yml]: Ups :/

**Comment by TomasVotruba** [comment]
Found a bug, back to WIP

**Comment by TomasVotruba** [comment]
Ok, now it's fixed (typo: evn/env).

When everything is ok, I can repush with clean commits.

Also I'd rename `lib` to `src`, since it's most used form.

**Comment by TomasVotruba** [comment]
Ping @Majkl578

**Comment by Majkl578** [comment]
@TomasVotruba: Hi, thanks for your great work, I rebased it pushed as 0d164572dec37f1ff5b33f01278828dc22de438c + 657455a8177f9d054a3fdd955cf79829d1fc9d52 + c60283d74c60241f415a59df3e10244fd56beb3e + b9bc64905b6b964ce956b836145d63f748edcbb3 + 109c1c72150e281d769d69bf53d5342f96939c66. Sorry for being late to the party. :)

**Comment by TomasVotruba** [comment]
@Majkl578 Thanks! I wonder if you still use Nette :)

**Pull request closed without merging**